poller_mikrotik.php

```diff
@@ -1256,7 +1256,7 @@ function collect_dhcp_details(&$host) {
 
 			cacti_log('MIKROTIK RouterOS API STATS: API Returned ' . sizeof($array) . ' DHCP Leases in ' . round($end-$start,2) . ' seconds.', false, 'SYSTEM');
 
-			if (sizeof($array)) {
+			if (sizeof($array) && !array_key_exists('!trap', $array)) {
```

Comment:
This seems generic, would it not also block other failure messages?

Reply:
OK, I'll run with it.  Since it is in the dhcp function, it should not affect anything else anyway.